README.md

```diff
@@ -1,79 +1,86 @@
 
 ![](gradients.png)
 
-If you wish to have subtle gradients with a nicely drawn separator for your titlebar, you can have just that. The gradient and separator will, like all nifty color features, use the title bar color to determine their values. How cool is that?
+> **Note**: if you wish to have subtle gradients with a nicely drawn separator for titlebars, you can have just that. The gradient and separator will, like all nifty color features, use the title bar color to determine their values. *How cool is that?*
 
 ## Experimental Features
+
+With the power of Halloween and the dark arts combined, we can now match the titlebars color to the windows color... And I don't mean some lame shit where I make use of `QPalette::Window` in the hopes of getting a nice looking window color... Hmmm, no, no no!
+
 ![](match_colors.png)
 
-With the power of Halloween and the dark arts combined, we can now match the titlebar color to the window color. And I don't mean some lame shit where I make use of ```QPalette::Window``` in the hopes of getting a nice looking window color, mmh no no no! This feature is still in early beta though and color glitches _will_ occur. Uncheck "Match colors" in the settings to turn the feature off.
+> **Note**: this feature is still in early beta though and color glitches *will* occur. Uncheck **Match colors** in Plasma Settings to turn this feature off.
 
-## Installation
-Clone the repository to a directory of your choice:
-```
-git clone https://github.com/n4n0GH/breezeway-mod
-```
-Note: it is recommended to clone it somewhere where you can leave the built directory in case you want to uninstall the theme. If you accidentally deleted the directory, check below for how to uninstall manually.
+## Building and Installation
 
-### Dependencies
+Before preceding with the actual install process, make sure you have the following dependencies installed (for Ubuntu 18.04 and derivates):
 
-Installation is currently only provided from source. Before preceding with the actual install process, make sure you have the following dependencies installed:
+* cmake
+* g++ / clang
+* extra-cmake-modules
+* libkdecorations2-dev
+* ibkf5guiaddons-dev
+* libkf5configwidgets-dev
+* libkf5windowsystem-dev
+* libkf5package-dev
+* libqt5x11extras5-dev
+* gettext
+* pkg-config *(optional)*
 
-#### Ubuntu / KDE Neon / Debian (probably)
+Clone this repository:
 
-1. cmake
-2. g++ / clang
-3. extra-cmake-modules
-4. libkdecorations2-dev
-5. libkf5guiaddons-dev
-6. libkf5configwidgets-dev
-7. libkf5windowsystem-dev
-8. libkf5package-dev
-9. libqt5x11extras5-dev
-10. gettext
-11. pkg-config (optional)
+```
+git clone https://github.com/n4n0GH/breezeway-mod
+```
+> **Note**: it's recommended to clone it somewhere where you can leave the built directory in case you want to uninstall it. If you accidentally deleted the directory, check below for how to uninstall manually.
```

Comment:
It should read `...where you can leave the build directory in case...`, I did that wrong as well.

Reply:
Fix it.